assets/cubyz/biomes/swamp/base.zig.zon

```diff
@@ -0,0 +1,48 @@
+.{
+	.properties = .{
+		.hot,
+		.wet,
+		.overgrown,
+	},
+	.minHeight = 12,
+	.maxHeight = 20,
```

Comment:
I think the swamp should be lower than that.
![Screenshot at 2025-01-15 17-20-24](https://github.com/user-attachments/assets/6f6badfd-9377-4181-87da-d9258af0126e)